test/OrchardCore.Tests/Localization/DefaultPluralRuleProviderTests.cs

```diff
@@ -25,5 +28,28 @@ public void TryGetRuleShouldReturnRuleForTopCulture(string culture, string expec
             Assert.NotNull(testPlural);
             Assert.Same(expectedPlural, testPlural);
         }
+
+        [Fact]
+        public async Task TestSayHello()
+        {
+            var context = new SiteContext();
+            await context.InitializeAsync();
+            await context.UsingTenantScopeAsync(async scope =>
+             {
+                 var shellFeaturesManager = scope.ServiceProvider.GetRequiredService<IShellFeaturesManager>();
+                 var availableFeatures = await shellFeaturesManager.GetAvailableFeaturesAsync();
+                 var demoFeature = availableFeatures.FirstOrDefault(feature => feature.Id == "OrchardCore.Demo");
+
+                 await shellFeaturesManager.EnableFeaturesAsync([demoFeature], true);
+
+                 var shellSettings = scope.ServiceProvider.GetRequiredService<ShellSettings>();
+                 var shellHost = scope.ServiceProvider.GetRequiredService<IShellHost>();
+                 await shellHost.ReloadShellContextAsync(shellSettings);
+             });
+
+            var response = await context.Client.GetAsync("api/demo/sayhello");
+
+            response.EnsureSuccessStatusCode();
```

Comment:
Hi @hishamco I'm trying to send a request here but it keeps returning 404 , I can't find out why, can you help me see what's going on?Thanks!

This PR is not meant to be merged into the main branch, it's just meant to replicate the translation issue

Reply:
Do you mean to reproduce the issue?

Reply:
> Do you mean to reproduce the issue?

yes